File: Web/src/test/java/forms/HelpForm.java
package forms;

import aquality.selenium.elements.interfaces.IButton;
import aquality.selenium.elements.interfaces.ILabel;
import aquality.selenium.forms.Form;
import org.openqa.selenium.By;

public class HelpForm extends Form {

    private final IButton hideForm = getElementFactory().getButton(By.xpath("//button[@class='button button--solid button--blue help-form__send-to-bottom-button']"),"hide form");
    private final ILabel isHidden = getElementFactory().getLabel(By.xpath("//div[@class='help-form is-hidden']"),"form is hidden");
    public HelpForm() {
        super(By.xpath("class='help-form__container'"), "help form");
    }

    public void hideHelpForm(){
        hideForm.click();
    }

    public boolean isHidden(){
        return isHidden.state().isExist();
    }

}

File: Web/TestProject1/FormClasses/HelpForm.cs
using Aquality.Selenium.Elements.Interfaces;
using Aquality.Selenium.Forms;
using OpenQA.Selenium;

namespace TestProject1
{
    class HelpForm : Form
    {
        public HelpForm() : base(By.XPath("//div[@class='help-form__container']//*[@class='button button--solid button--blue help-form__close-button']"), "Help Form")
        {
        }
        public IButton SendDownButton => ElementFactory.GetButton(By.XPath("//div[@class='align__cell u-right']//*[@class='button button--solid button--blue help-form__send-to-bottom-button']"), "Send down Button");

        public void SendFormDown()
        {
            SendDownButton.Click();
        }
    }
}
